hw/top_earlgrey/ip/pinmux/doc/autogen/targets.md

```diff
@@ -4,8 +4,8 @@ DO NOT EDIT THIS FILE DIRECTLY.
 It has been generated with the following command:
 util/topgen.py -t hw/top_earlgrey/data/top_earlgrey.hjson -o hw/top_earlgrey/
 -->
-|  Target Name  |  #IO Banks  |  #Muxed Pads  |  #Direct Pads  |  #Manual Pads  |  #Total Pads  |      Pinout / Pinmux Tables       |
-|:-------------:|:-----------:|:-------------:|:--------------:|:--------------:|:-------------:|:---------------------------------:|
-|     ASIC      |      4      |      47       |       14       |       10       |      71       | [Pinout Table](./pinout_asic.md)  |
-|     CW310     |      4      |      47       |       14       |       14       |      75       | [Pinout Table](./pinout_cw310.md) |
-|     CW340     |      4      |      47       |       14       |       13       |      74       | [Pinout Table](./pinout_cw340.md) |
+|  Target Name  |  #IO Banks  |  #Muxed Pads  |  #Direct Pads  |  #Manual Pads  |  #Total Pads  |                           Pinout / Pinmux Tables                            |
+|:-------------:|:-----------:|:-------------:|:--------------:|:--------------:|:-------------:|:---------------------------------------------------------------------------:|
+|     ASIC      |      4      |      47       |       14       |       10       |      71       | [Pinout Table](../../../top_earlgrey/ip/pinmux/doc/autogen/pinout_asic.md)  |
```

Comment:
Hm... actually, is this the right number of directory changes? It looks to me like this new link wouldn't resolve.

Also, did you find a broken link on the website? It was working for me.

Reply:
It seems like this would break the top-level documentation here: https://opentitan.org/book/hw/top_earlgrey/ip/pinmux/doc/autogen/targets.html

Reply:
There are two scripts, I believe. One that generates top-level docs, and one for the IP itself. The latter is broken:
https://opentitan.org/book/hw/ip/pinmux/doc/programmers_guide.html#top_earlgrey-pinmux-targets

Reply:
Ah I see now that this table is reused here:
https://opentitan.org/book/hw/top_earlgrey/ip/pinmux/doc/autogen/targets.html

That was not how this was initially intended.

Reply:
I'm feeling like the top-level documentation is already fine, but it's the IP-level that is broken (which is referenced in the issue you linked). Should we consider eliminating the top-level-specific pin-out table from the IP docs instead?

https://github.com/lowRISC/opentitan/blob/c3d9a993d78670d45f9085466e80fe6d263d829e/hw/ip/pinmux/doc/programmers_guide.md?plain=1#L109-L115

Reply:
Yeah that may be the easier fix. Let me do that.

Reply:
Updated - PTAL